README.md

```diff
@@ -83,12 +83,17 @@ Unix Build Instructions
 
 Run
 ---
-Adapt the configuration file ``/etc/mympd.conf`` to your needs.
+Adapt the configuration file ``/etc/webapps/mympd/mympd.conf`` to your needs.
```

Comment:
``/etc/webapps/mympd/mympd.conf`` is only for arch, the default is ``/etc/mympd.conf``